mojo/dart/packages/mojo/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.4.17
+
+  - 48 changes: https://github.com/domokit/mojo/compare/2829804...1484e83
+
+  MOJO_SDK: 1484e831758d05c7af0f27c199d24e3bbdc6383f
```

Comment:
Maybe your comment wasn't specific to this file and you just meant in general that I would be nice if we annotated the base mojo revision.  I'm happy to do that.